docs/book/src/view/01_basic_component.md

```diff
@@ -57,7 +57,7 @@ Every component is a function with the following characteristics
 1. It takes a reactive [`Scope`](https://docs.rs/leptos/latest/leptos/struct.Scope.html)
    as its first argument. This `Scope` is our entrypoint into the reactive system.
    By convention, it’s usually named `cx`.
-2. You can include other arguments, which will be available as component “props.”
+2. You can include other arguments, which will be available as component “props”.
```

Comment:
Position of punctuation and quotation marks is always up for debate but "punctuation inside quotation marks" is [the default in the United States](https://style.mla.org/punctuation-and-quotation-marks/). I think they're quite consistent throughout the book.